acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/Eth2RequestUtils.java

```diff
@@ -473,7 +486,38 @@ private static Eth2SigningRequestBody createSyncCommitteeContributionAndProofReq
         null,
         null,
         null,
-        getContributionAndProof());
+        getContributionAndProof(),
+        null);
+  }
+
+  private static Eth2SigningRequestBody createRegisterValidatorRequest() {
+    final UInt64 epoch = dataStructureUtil.randomEpoch();
+    final ValidatorRegistration validatorRegistration =
+        new ValidatorRegistration(
+            dataStructureUtil.randomBytes20(),
+            dataStructureUtil.randomUInt64(),
+            dataStructureUtil.randomUInt64(),
+            BLSPubKey.fromHexString(
+                "0x8f82597c919c056571a05dfe83e6a7d32acf9ad8931be04d11384e95468cd68b40129864ae12745f774654bbac09b057"));
```

Comment:
nit: be nice to create a constant for this public key since we use in at least two create request methods